src/StarterGui/WesTimingGui/TimingLocalScript/TimingKeeperService.lua

```diff
@@ -40,7 +40,35 @@ function service:HandleSector(sector: number)
     elseif sector == 2 then
         sectorTime = time() - sector1At
         sector2At = time()
-    end
+	end
+	
+	-- Get delta time data
+	if _config.SectorTimeDelta == true then
```

Comment:
`== true` is redundant.

```suggestion
	if _config.SectorTimeDelta then
```